e107_admin/includes/infopanel.php

```diff
@@ -771,27 +768,32 @@ private function getStats($type)
 		{
 			$data = array();
 		
-			$data['labels'] 	= array("January","February","March","April","May","June","July");
-			
+			$data['labels']	= array(strftime("%B",mktime(0,0,0,01,01,16)), //"January",
+							strftime("%B",mktime(0,0,0,02,01,16)), //"February",
+							strftime("%B",mktime(0,0,0,03,01,16)), //"March",
+							strftime("%B",mktime(0,0,0,04,01,16)), //"April",
+							strftime("%B",mktime(0,0,0,05,01,16)), //"May",
+							strftime("%B",mktime(0,0,0,06,01,16)), //"June",
+							strftime("%B",mktime(0,0,0,07,01,16))  //"July"
+			);
```

Comment:
There's an easier way for this. 

    $months = e107::getDate()->terms(); 
    print_a($months);